app/src/main/java/org/oppia/android/app/walkthrough/ShowHideProgressBarAndHeaderTextview.kt

```diff
@@ -0,0 +1,6 @@
+package org.oppia.android.app.walkthrough
+
+interface ShowHideProgressBarAndHeaderTextview {
```

Comment:
As per the naming convention, all our listeners should be ending with `*Listeners.kt`. Please follow the same. 

@rt4914 I think the name should be `WalkThroughVerticalScrollListener`. What do you think on this?

Reply:
Done

Reply:
Sounds good.